ScanCommand/src/test/java/io/ballerina/scan/ScanCmdTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.scan;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import picocli.CommandLine;
+
+import java.io.IOException;
+
+public class ScanCmdTest extends ScanCommandTest {
+    @Test(description = "test scan command")
+    void testScanCommand() throws IOException {
+        ScanCmd scanCmd = new ScanCmd(outputStream);
+        scanCmd.execute();
+        String scanLog = readOutput(true);
+        Assert.assertEquals("Running Scans...", scanLog.trim().split("\n")[0]);
+    }
+
+    @Test(description = "test scan command with help flag")
+    void testScanCommandWithHelpFlag() throws IOException {
+        ScanCmd scanCmd = new ScanCmd(outputStream);
+        String[] args = {"--help"};
+        new CommandLine(scanCmd).parseArgs(args);
+        scanCmd.execute();
+        String scanLog = readOutput(true);
+        Assert.assertTrue(scanLog.contains("NAME\n" + "        ballerina-scan - Static code analyzer"));
```

Comment:
Can't we assert the entire text?

Reply:
This is possible however, the test configurations follow the pattern similar to [other Ballerina commands](https://github.com/ballerina-platform/openapi-tools/blob/master/openapi-cli/src/test/java/io/ballerina/openapi/cmd/OpenAPICmdTest.java#L70)

Reply:
If there are potential improvements, we should introduce them. Irrespective of what other commands do.